### DISCUSSION TITLE
export to HTML doesn't open easy in Excel anymore after update

### DISCUSSION THREAD

**Discussion by denmusti** [dbeaver/dbeaver#13845] [Q&A] [answered]
#### System information: 
- Windows 10
- DBeaver version: Latest

#### Describe the problem you're observing:
Because we have to distribute a large portion of data (200K rows but lots of columns) to our internal customers we export the data to HTML.
After that we copy it to Excel in a template as values so all values are shown correctly without any typical excel 'enhancements'.
Normally when we open the HTML -exportfile in Excel it takes a couple of minutes (the file is +300MB) and we get the data with the query in a topcell  spanning over all rows. the result is an 25MB Excelfile

But since yesterday after i update DBeaver to the latest version, opening the HTML export in Excel takes almost 2 hours.
At first it then looks like i'ts empty but the query is in white spread over  several rows ( = number of rows the query is long)
Underneath is the data we need intact.

how can this be fixed so Excel will open the HTML exports fast again an the query i shown as it was.
Not showing the query is also an option but i haven't found that functionallity yet

**Comment by p91paul** [comment, marked as answer]
Have you tried exporting directly to excel? there is an office plugin available.

**Comment by denmusti** [comment]
@p91paul where can i find that one?
Although we allways save als xlsb-files beacause of the mass load of data.

**Comment by p91paul** [comment]
https://github.com/dbeaver/dbeaver/wiki/Optional-extensions
